src/cli/cmds/init.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import * as fs from 'fs';
+import Color = require('color');
+import * as inquirer from 'inquirer';
+import {Config} from '../../lib/Config';
+import {JdkHelper} from '../../lib/jdk/JdkHelper';
+import {KeyTool} from '../../lib/jdk/KeyTool';
+import Log from '../../lib/Log';
+import {TwaGenerator} from '../../lib/TwaGenerator';
+import {TwaManifest} from '../../lib/TwaManifest';
+import {validateColor, validatePassword, validateUrl, notEmpty} from '../inputHelpers';
+import {ParsedArgs} from 'minimist';
+
+const log = new Log('init');
+
+export interface InitArgs {
+  manifest: string;
+  directory?: string;
+}
+
+async function confirmTwaConfig(twaManifest: TwaManifest): Promise<TwaManifest> {
+  const result = await inquirer.prompt([
+    {
+      name: 'host',
+      type: 'input',
+      message: 'Domain being opened in the TWA:',
+      default: twaManifest.host,
+      validate: notEmpty,
+    }, {
+      name: 'name',
+      type: 'input',
+      message: 'Name of the application:',
+      default: twaManifest.name,
+      validate: notEmpty,
```

Comment:
Similarly, we could check that this and `launcherName` aren't too long.

Reply:
Another good one for #31 (Adding here so I get the mentions in the issue)